esiclient/tests/v1/test_trunk.py

```diff
@@ -328,3 +330,265 @@ def test_take_action(self):
                 mock.call("port_uuid_3"),
                 mock.call("port_uuid_1"),
             ])
+
+
+class TestAddNetwork(base.TestCommand):
+
+    def setUp(self):
+        super(TestAddNetwork, self).setUp()
+        self.cmd = trunk.AddNetwork(self.app, None)
+
+        self.network2 = utils.create_mock_object({
+            "id": "network_uuid_2",
+            "name": "network2",
+            "provider_segmentation_id": 222
+        })
+        self.network3 = utils.create_mock_object({
+            "id": "network_uuid_3",
+            "name": "network3",
+            "provider_segmentation_id": 333
+        })
+        self.subport1 = utils.create_mock_object({
```

Comment:
Maybe name the subports as subport2 and subport3 would align with the ports and networks id?

Reply:
Fair enough! Should be fixed now.